servicenow/README.md

```diff
@@ -0,0 +1,184 @@
+# **ServiceNow Integration**
+
+
+## 
+**Setup**
+
+
+### 
+**Prerequisites**
+
+
+
+*   Follow the instructions on the [main README file](https://github.com/GoogleCloudPlatform/dialogflow-integrations#readme) in the root directory of this repository.
```

Comment:
I'm not sure if they still need to follow main README for this case. As main README is also instructing to set up a Cloud Run instance. Do you need a Cloud Run instance for ServiceNow integration?

Reply:
There is no Cloud Run setup for this.  I'll remove.